src/index.js

```diff
@@ -20,12 +20,30 @@ const client = new ApolloClient({
   cache: new InMemoryCache(),
 });
 
-ReactDOM.render(
-  <ApolloProvider client={client}>
-    <App />
-  </ApolloProvider>,
-  document.getElementById('root')
-);
+function renderApp(config) {
+  ReactDOM.render(
+    <ApolloProvider client={client}>
+      <App {...config} />
+    </ApolloProvider>,
+    document.getElementById('root')
+  );
+}
+
+fetch('config.json')
+  .then(response => {
+    if (!response.ok) {
+      throw Error('Got ' + response.status + ' http status');
+    }
+    return response.json();
+  })
+  .then(config => renderApp(config))
```

Comment:
As @fedde-s  noted it could be `.then(renderApp)`

Reply:
You inlined it; this one's resolved as well :heavy_check_mark: